hist/histpainter/src/THistPainter.cxx

```diff
@@ -2604,7 +2604,7 @@ Begin_Macro(source)
       z = x*x + y*y;
       h3scat->Fill(x,y,z);
    }
-   h3scat->Draw();
+   h3scat->Draw("SCAT");
```

Comment:
I remember some discussion around this unfortunate choice of wording for this histogram painting option. IIRC the better alternative is `TScatter` in this case, isn't it? Maybe we can consider removing this legacy example with a bad word choice and just redirect to the TScatter docs now?

Reply:
It is not the promotion of this option. It is still deprecated. BUT, it is also still documented and, because the default option is now COL, this example was not correct as it was claiming being a SCAT example whereas it shows a COL plot. That's only that, not more. It is just a matter of coherence. We recently had a question on the forum from somebody noticing the change of default. So I guess it is not yet the time to remove this option. It is there, not default, and can be used with all the warnings in the doc.

Reply:
One idea: just adding a comment in the code instead